test/minikube/001-kube.bats

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the minikube cluster before the first test
+    if [ "$BATS_TEST_NUMBER" -eq 1 ]; then
+        minikube start
+        echo $output >&2
+    fi
+    # wait for a bit so all the services in the cluster come up
+    sleep 10
+}
+
+###############################################################################
+# BEGIN tests
+
+@test "minikube - check cluster is up" {
+    run minikube kubectl get nodes
+    [ "$status" -eq 0 ]
+    [[ "$output" == *"Ready"* ]]
+    run minikube kubectl get pods
+    [ "$status" -eq 0 ]
+    [[ "$output" == "No resources found in default namespace." ]]
```

Comment:
(Caveat: I'm not a bats expert)
Minor nit: in bash, `[` is normally an alias for `/bin/test` where as `[[` is a bash built-in.  It's slightly confusing to see the two being used together like this without a comment explaining why.  It would be good to choose one and stick with it.  Or, if there's a good reason for deviating, add a comment so others understand why.

Reply:
No reason, switched all to `[[`